.github/workflows/autobuild.yml

```diff
@@ -45,29 +42,25 @@ jobs:
 
 
      steps:
-         # Checkout code
          - name:                    Checkout code
            uses:                    actions/checkout@v2
 
-         # Set variables
-         # Determine release / pre-release
-         - name:                    Get Jamulus build info, determine actions & variables
+         - name:                    Determine release version, type and prerelease variables
            run:                     python3 ${{ github.workspace }}/.github/actions_scripts/analyse_git_reference.py
            id:                      jamulus-build-vars
 
-         # remove release, if it exists (with this releasetag)
```

Comment:
I think this was mainly a bug-fix or related to pushing a release tag? Not sure why this was needed again (ie. why we needed to check for an existing release. But it's good to have it anyways.)

Reply:
Might also be relevant if we encounter any issues with the PR run and have to re-start it. I'd keep it (especially as part of this PR).